monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,115 @@
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.6/storage/understanding-persistent-storage.html[Understanding persistent storage]
+* link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/optimizing-storage.html[Optimizing storage]
+
+// Managing scrape sample limits for user-defined projects
+include::modules/osd-monitoring-limiting-scrape-samples-in-user-defined-projects.adoc[leveloffset=+1]
+include::modules/osd-monitoring-setting-a-scrape-sample-limit-for-user-defined-projects.adoc[leveloffset=+2]
+include::modules/osd-monitoring-creating-scrape-sample-alerts.adoc[leveloffset=+2]
+
+.Additional resources
+
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.6/monitoring/troubleshooting-monitoring-issues.html#determining-why-prometheus-is-consuming-disk-space_troubleshooting-monitoring-issues[Determining why Prometheus is consuming a lot of disk space] for steps to query which metrics have the highest number of scrape samples
+
+//Attaching additional labels to your time series and alerts
+include::modules/monitoring-attaching-additional-labels-to-your-time-series-and-alerts.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+
+// Setting log levels for monitoring components
+include::modules/osd-monitoring-setting-log-levels-for-monitoring-components.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+
+== Next steps
+
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+* Learn about link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html[remote health reporting] and, if necessary, opt out of it
```

Comment:
This is a link that directs the reader to the OpenShift Container Platform documentation. Is this relevant to ROSA, or should this link be removed? The reader might not realise that they are being directed to the docs for a different platform type.

Reply:
Keep.